INSTALL.md

```diff
@@ -0,0 +1,86 @@
+# Installing requirements
+
+## Mandatory requirements
+
+  * Python 2.7 - the code here is not Python 3.x ready
+  * bzip2 / bunzip2 and bzip2 (including header files)
+  * curl (including header files)
+  * gcc and make
+  * gfortran (including header files)
+  * liblzma library (including header files)
+  * Python modules: numpy, pip, libcap
+  * openssl (including header files)
+  * pkg-config
+  * pcre library (including header files)
+  * readline (including header files)
+  * wget
+  * xorg (including header files)
```

Comment:
Do we definitely need this for warmup_stats? I'm sorta surprised...

Reply:
TBH I can't remember what needed it, but it was a dependency of something or other.

Reply:
I think that was for the main experiment, so I'm inclined to drop it, because it can be an onerous requirement for some servers. [I *think* at least one person has built without xorg. I don't know if we can test this ourselves though.]